types/node/fs.d.ts

```diff
@@ -275,6 +275,7 @@ declare module "fs" {
     /**
      * Asynchronous lchown(2) - Change ownership of a file. Does not dereference symbolic links.
      * @param path A path to a file. If a URL is provided, it must use the `file:` protocol.
+     * @deprecated
```

Comment:
according to [docs](https://nodejs.org/dist/latest-v11.x/docs/api/fs.html#fs_fs_lchown_path_uid_gid_callback) `lchown` and `lchownSync` are no longer deprecated since 10.6.0